src/components/filter/filter.tsx

```diff
@@ -158,7 +158,7 @@ export class Filter implements InteractiveComponent {
           if (find(val, RE)) {
             found = true;
           }
-        } else if (RE.test(val)) {
+        } else if (typeof val === "string" && RE.test(val)) {
           found = true;
```

Comment:
@jcfranco should this also accept `number` typed values? Maybe if there is a rating property or something that people want to filter by

Reply:
The current version changes the filtering behavior, right? I think pick/value list items allow passing metadata objects that go through filtering.

How about returning if the value is `null` or `undefined` instead?

```ts
if (typeof val === "function" || val == null /* intentional == to catch undefined */) {
  return;
}

if (Array.isArray(val) || typeof val === "object") { /* ... */ }
{ else if (RE.test(val)) { /* ... */ }
```